client/my-sites/sidebar/static-data/all-sites-menu.js

```diff
@@ -12,41 +12,47 @@ export default function allSitesMenu() {
 			icon: 'dashicons-chart-bar',
 			slug: 'stats',
 			title: translate( 'Stats' ),
+			navigationLabel: translate( 'View stats for all sites' ),
```

Comment:
I've added individual translations for each of the "all sites" links. Previously we were using string interpolation and assuming it'd be to just make the `title` lowercase. I'm not sure but that seemed a bit English-centric. Maybe there's some gendered-ness or something else that'll mean the title can't just be inserted in the middle of a sentence.